src/cdf.rs

```diff
@@ -753,699 +753,678 @@ const fn cdf3d<const P: usize, const N: usize, const M: usize, const L: usize, c
     }
     cdf3d
 }
-pub const fn av1_default_cdf() -> CdfModeContext {
-    let mut init = CdfModeContext {
-        y_mode: Align32(cdf1d([
+static mut av1_default_cdf: CdfModeContext = CdfModeContext {
```

Comment:
Why do we need/want `static mut`s here instead of a `const fn`?  `static mut`s are `unsafe` to access.

Reply:
I was simply matching existing code patterns. I'm happy to drop the `mut` from all of the constants.